public/pages/Main/Main.tsx

```diff
@@ -171,23 +171,29 @@ export default class Main extends Component<MainProps, object> {
                           <Route
                             path={ROUTES.SNAPSHOTS}
                             render={(props: RouteComponentProps) => (
-                              <Snapshots
-                                {...props}
-                                snapshotManagementService={services.snapshotManagementService}
-                                indexService={services.indexService}
-                              />
+                              <div style={{ padding: "25px 25px" }}>
```

Comment:
Nitpick. It looks like these padding values are reused for all of these routes. If the intent is to have a consistent style for all routes, it may be worthwhile to instead implement a constant (e.g., `const ROUTE_STYLE = { padding: "25px 25px" }`) to use instead of setting the styles individually.

Reply:
Yes, I will include screenshots.  And yes, I do agree a constant would be more dry.  Thank you!

Reply:
Good point about updating the snapshots as well.  Thanks again!

Reply:
All changes made in response to comments.  Screenshot compilations below.  Apologies if they aren't displaying inline.